authors/apps/article_likes/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from authors.apps.authentication.models import User
+from authors.apps.articles.models import Article
+
+class ArticleLike(models.Model):
+    author = models.ForeignKey(User, to_field="username", on_delete=models.CASCADE)
```

Comment:
When you say author, are you referring to the author of the article or the user liking the article? If it is the latter, then author is a confusing attribute name.